public class com.android.internal.logging.AndroidConfig extends java.lang.Object
{

    public void <init>()
    {
        com.android.internal.logging.AndroidConfig r0;
        java.util.logging.Logger r1, $r3;
        java.util.logging.Level $r2, $r4;
        java.lang.Exception $r5;
        com.android.internal.logging.AndroidHandler r6;

        r0 := @this: com.android.internal.logging.AndroidConfig;

        specialinvoke r0.<java.lang.Object: void <init>()>();

     label1:
        r1 = staticinvoke <java.util.logging.Logger: java.util.logging.Logger getLogger(java.lang.String)>("");

        r6 = new com.android.internal.logging.AndroidHandler;

        specialinvoke r6.<com.android.internal.logging.AndroidHandler: void <init>()>();

        virtualinvoke r1.<java.util.logging.Logger: void addHandler(java.util.logging.Handler)>(r6);

        $r2 = <java.util.logging.Level: java.util.logging.Level INFO>;

        virtualinvoke r1.<java.util.logging.Logger: void setLevel(java.util.logging.Level)>($r2);

        $r3 = staticinvoke <java.util.logging.Logger: java.util.logging.Logger getLogger(java.lang.String)>("org.apache");

        $r4 = <java.util.logging.Level: java.util.logging.Level WARNING>;

        virtualinvoke $r3.<java.util.logging.Logger: void setLevel(java.util.logging.Level)>($r4);

     label2:
        goto label4;

     label3:
        $r5 := @caughtexception;

        virtualinvoke $r5.<java.lang.Exception: void printStackTrace()>();

     label4:
        return;

        catch java.lang.Exception from label1 to label2 with label3;
    }
}
